### PR TITLE
Add customization section with instructions to configure GA

### LINKED ISSUE #108: Document how to add custom GA analytics (for developers)

### PR DESCRIPTION
**Pull request by delapuente**
Fix #108

### CODE REVIEW

**Comment by cvan** [comment]
@delapuente: you absolutely made my day! I had this next up on my to-do list. this is very thorough and helpful. (will review properly in a few.)

if I may add: I like that you are holding the developers' hands through these flows; especially at this stage (when we're trying to get adoption), I think it's the best possible thing to do.

P.S. lovely screenshots.

File: docs/customization/adding-ga.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+![Account and website names, website URL, industry classifications and reporting timezone](./images/filled-1.png)
+
+Whe you finish, click on `Get Tracking ID` at the bottom of the page:
+
+![Buttons Get Tracking ID and Cancel at the bottom of the page](./images/filled-2.png)
+
+This will redirect you to the configuration page where you find the HTML code you need to use in the template:
+
+![The configuraiton page includes the instructions for setting up your site](./images/setup-done.png)
```
Comment (cvan) [inline docs/customization/adding-ga.md]: minor typo: `configuraiton`

File: docs/customization/adding-ga.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Now, in your project, with your favourite code editor, open the file at `Assets/WebGLTemplates/WebVR/index.html` and paste the code provided by Google Analytics just after the `<head>` tag:
+
+![The code provided by GA should be added right after the head tag](./images/add-to-index.png)
+
+## Getting started
+
+The amount of data in Google Analytics is huge and overwhelming. Fortunately, the folks in Google have developed a series of courses to explain you the main concepts of traffic analysicis. We recommend you start at [Google Analytics Academy](https://analytics.google.com/analytics/academy/).
```
Comment (cvan) [inline docs/customization/adding-ga.md]: typo: `analysicis`

File: docs/customization/adding-ga.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+![The code provided by GA should be added right after the head tag](./images/add-to-index.png)
+
+## Getting started
+
+The amount of data in Google Analytics is huge and overwhelming. Fortunately, the folks in Google have developed a series of courses to explain you the main concepts of traffic analysicis. We recommend you start at [Google Analytics Academy](https://analytics.google.com/analytics/academy/). 
+
+## Free/Open source alternatives
+
+There are a few [free/open source alternatives](https://en.wikipedia.org/wiki/List_of_web_analytics_software#Free_/_Open_source_(FLOSS)) out there. We recommend [Matomo](https://matomo.org/) and [Open Web Analytics](http://www.openwebanalytics.com) (OWA). Both offer geolocation, click heat maps, traffic by referrer, event tracking... Which specifically depends on your needs but you can compare both side by side visiting the [demo pages for Motomo](https://demo.matomo.org) and [OWA](http://demo.openwebanalytics.com/owa/).
```
Comment (cvan) [inline docs/customization/adding-ga.md]: `open source` -> `open-source`

File: docs/customization/adding-ga.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+You will need to modify the VR template that comes with the package so be sure you've already added [the package from the store](http://u3d.as/1476) to your Unity project.
+
+## Set up
+
+Start by visiting [Google Analytics](https://analytics.google.com/analytics/web) and sign up for a new account:
+
+![Summary of the three steps to setup Google Analytics](./images/setup-ga.png)
+
+Fill in the details of your organization and website. Remember to select `https` in the drop menu of the `Website URL` field.
```
Comment (cvan) [inline docs/customization/adding-ga.md]: `https://`

Comment (cvan) [reply docs/customization/adding-ga.md]: `drop menu` -> `drop-drop menu`?

File: docs/customization/adding-ga.md (hunk 2 above)
Comment (cvan) [inline docs/customization/adding-ga.md]: `folks in Google` -> `folks at Google`

File: docs/customization/adding-ga.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# How to add Google Analytics to your game
+
+Google Analytics is a popular web analytics service by Google that tracks and reports website traffic. It offers insight about your user behaviour and allow you to make informed decisions.
```
Comment (cvan) [inline docs/customization/adding-ga.md]: can remove the ` by Google` part (redundant)

Comment (cvan) [reply docs/customization/adding-ga.md]: `decisions`? about development? improving your game/page?

File: docs/customization/adding-ga.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+You will need to modify the VR template that comes with the package so be sure you've already added [the package from the store](http://u3d.as/1476) to your Unity project.
```
Comment (cvan) [inline docs/customization/adding-ga.md]: can we link `VR template` and/or reference to the filename of the `index.html` HTML file?

Comment (cvan) [reply docs/customization/adding-ga.md]: `so be sure` -> `, so be sure`

File: docs/customization/adding-ga.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+Start by visiting [Google Analytics](https://analytics.google.com/analytics/web) and sign up for a new account:
+
+![Summary of the three steps to setup Google Analytics](./images/setup-ga.png)
+
+Fill in the details of your organization and website. Remember to select `https` in the drop menu of the `Website URL` field.
+
+![Account and website names, website URL, industry classifications and reporting timezone](./images/filled-1.png)
+
+Whe you finish, click on `Get Tracking ID` at the bottom of the page:
```
Comment (cvan) [inline docs/customization/adding-ga.md]: typo: `Whe` -> `When`

File: docs/customization/adding-ga.md (hunk 2 above)
Comment (cvan) [inline docs/customization/adding-ga.md]: suggestion: `The amount of data in Google Analytics is huge and overwhelming.` -> `The amount of data in Google Analytics can be overwhelming to navigate and dissect.` (or something like that?)

File: docs/customization/adding-ga.md (hunk 3 above)
Comment (cvan) [inline docs/customization/adding-ga.md]: `click heat maps` -> `click-heat map`

Comment (cvan) [reply docs/customization/adding-ga.md]: `...` -> `, and so on` or `, etc.`?

Comment (cvan) [reply docs/customization/adding-ga.md]: typo: `Motomo` -> `Matomo`

Comment (cvan) [inline docs/customization/adding-ga.md]: nice addition here! I've never used these two solutions, but the demo pages look quite good 👍

Comment (cvan) [reply docs/customization/adding-ga.md]: small nit/question: Google Analytics is technically also free; it's just not a host-your-self + open-source solution. `alternatives` is the correct word. you don't think that it'd be inferred that Google Analytics is paid? (ignore my question here if you think it's unambiguous.)

Comment (delapuente) [reply docs/customization/adding-ga.md]: Agree. I changed the title to simply `Open-Source alternatives`.

**Comment by delapuente** [comment]
This is done, @cvan do you mind to review again?

File: docs/customization/adding-ga.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
+# How to add Google Analytics to your game
+
+Google Analytics is a popular web analytics service that tracks and reports website traffic. It offers insight about your user behaviour and allow you to make informed decisions regarding the evolution of your site.
```
Comment (cvan) [inline docs/customization/adding-ga.md]: `allow` -> `allows`

File: docs/customization/adding-ga.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+You will need to modify the [VR template](../../Assets/WebGLTemplates/WebVR/index.html) that comes with the package, so be sure you've already added [the package from the store](http://u3d.as/1476) to your Unity project.
```
Comment (cvan) [inline docs/customization/adding-ga.md]: `https://u3d.as/1476` works too

File: docs/customization/adding-ga.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+
+When you finish, click on `Get Tracking ID` at the bottom of the page:
+
+![Buttons Get Tracking ID and Cancel at the bottom of the page](./images/filled-2.png)
+
+This will redirect you to the configuration page where you find the HTML code you need to use in the template:
+
+![The configuration page includes the instructions for setting up your site](./images/setup-done.png)
+
+Now, in your project, with your favourite code editor, open the file at `Assets/WebGLTemplates/WebVR/index.html` and paste the code provided by Google Analytics just after the `<head>` tag:
```
Comment (cvan) [inline docs/customization/adding-ga.md]: `just after the` -> technically `inside, preferably at the top of, the`? (or a more succinct version of that)

File: docs/customization/adding-ga.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+
+![The code provided by GA should be added right after the head tag](./images/add-to-index.png)
+
+## Getting started
+
+The amount of data in Google Analytics can be overwhelming to navigate and dissect. Fortunately, the folks in Google have developed a series of courses to explain you the main concepts of traffic analysis. We recommend you start at [Google Analytics Academy](https://analytics.google.com/analytics/academy/).
```
Comment (cvan) [inline docs/customization/adding-ga.md]: `in Google` -> `at Google`

File: docs/customization/adding-ga.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+
+![The code provided by GA should be added right after the head tag](./images/add-to-index.png)
+
+## Getting started
+
+The amount of data in Google Analytics can be overwhelming to navigate and dissect. Fortunately, the folks in Google have developed a series of courses to explain you the main concepts of traffic analysis. We recommend you start at [Google Analytics Academy](https://analytics.google.com/analytics/academy/). 
+
+## Open-source alternatives
+
+There are a few [open-source alternatives](https://en.wikipedia.org/wiki/List_of_web_analytics_software#Free_/_Open_source_(FLOSS)) out there. Like Google Analytics, they are free although you need to host them yourself. We recommend [Matomo](https://matomo.org/) and [Open Web Analytics](http://www.openwebanalytics.com) (OWA). Both offer geolocation, click-heat maps, traffic by referrer, event tracking, etc. Which specifically depends on your needs but you can compare both side by side visiting the [demo pages for Matomo](https://demo.matomo.org) and [OWA](http://demo.openwebanalytics.com/owa/).
```
Comment (cvan) [inline docs/customization/adding-ga.md]: nice changes here. this paragraph reads well, and it's informative 👍

File: docs/customization/adding-ga.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+
+This will redirect you to the configuration page where you find the HTML code you need to use in the template:
+
+![The configuration page includes the instructions for setting up your site](./images/setup-done.png)
+
+Now, in your project, with your favourite code editor, open the file at `Assets/WebGLTemplates/WebVR/index.html` and paste the code provided by Google Analytics just after the `<head>` tag:
+
+![The code provided by GA should be added right after the head tag](./images/add-to-index.png)
+
+## Getting started
```
Comment (cvan) [inline docs/customization/adding-ga.md]: do you plan to add more info. to this heading? if not, perhaps a heading such as `## Using Google Analytics` would be more aptly named?

**Comment by delapuente** [comment]
Done, I've quantified the PNGs also. I'm going to merge this.

**Pull request merged**